components/dev/UiKit/Icons.tsx

```diff
@@ -0,0 +1,54 @@
+import { PageWrapper } from 'components/PageWrapper/PageWrapper'
+import style from '@/@ui/ui-kit/Icon/IconsComponent.module.scss'
+import { Icons } from '@/@ui/ui-kit/Icon/IconsComponent'
+
+export const getStaticProps = async () => {
+  return {
+    props: {},
+  }
+}
+
+const IconsDev = () => {
+  return (
+    <PageWrapper>
+      <div className={style.kitContainer}>
+
+        <div style={{ display: 'flex', textAlign: 'center', color: 'white' }}>
+        <div>
+          <h2>Default</h2>
+          <Icons.Home className={style.SideBar__Item} />
```

Comment:
it's inconvenient to specify a classname each time when creating an icon

Reply:
done